ghastly/ghastly.py

```diff
@@ -0,0 +1,292 @@
+import numpy as np
+import openmc
+import ghastly
+from jinja2 import Environment, PackageLoader
+from ghastly import read_input
+from ghastly import region
+from lammps import lammps
+
+env = Environment(loader=PackageLoader('ghastly','templates'))
+
+def fill_core(input_file, rough_pf):
+    '''
+    This function uses OpenMC's model.pack_spheres function to "rough" pack
+    the full core region defined by the Ghastly input file, then uses a series
+    of Jinja templates to create a LAMMPS input file that will run a physics
+    simulation, modeling not only the settling of the roughly packed pebbles
+    due to gravtiy, but also handle the addtion of any extra pebbles needed to
+    reach the target number of pebbles needed.
+
+    Parameters
+    ----------
+    input_file : str
+        Name of ghastly input file, which should be in JSON format, following
+        the layout and parameter naming convention given in the example inputs.
+    rough_pf : float
+        Packing fraction used in OpenMC packing function.  Can match the target
+        packing fraction provided in the input file, but users should take care 
+        supplying a rough_pf that is greater than the input's target pf.
+    '''
+
+    input_block = read_input.InputBlock(input_file)
+    sim_block = input_block.create_obj()
+
+    rough_pack = []
+    core_volume = 0
+    pack_zones = sim_block.core_main | sim_block.core_outtake
+    for element in pack_zones.values():
+        core_volume += element.volume
+        if type(element) == ghastly.core.CylCore:
+            coords = pack_cyl(sim_block, element, rough_pf)
+            rough_pack += coords
+        else:
+            pass
+
+    n_pebbles = int((sim_block.pf*core_volume)/sim_block.pebble_volume)
+    print("Equivalent number of pebbles is "+str(n_pebbles))
+
+    pebbles_left = n_pebbles - len(rough_pack)
+
+    if pebbles_left < 0.0:
```

Comment:
```suggestion
    if pebbles_left < 0:
```
Since this should always be an integer